mobile/src/components/course/course.tsx

```diff
@@ -42,6 +114,24 @@ const Course: FC = () => {
     <ScrollView>
       <View style={styles.container}>
         <Text style={styles.h1}>{dataCourse?.title}</Text>
+        <View style={styles.dropdownContainer}>
+          {editMode ? (
+            <Dropdown
+              items={categories}
+              onSelectItem={(id): Promise<void> =>
+                handleSubmit(() => handleSelectNewCategory(id))()
+              }
```

Comment:
It shouldn't submit something
```suggestion
              onSelectItem={handleSelectNewCategory}
              }
```